morf-core/src/main/java/org/alfasoftware/morf/sql/element/TableReference.java

```diff
@@ -52,6 +52,12 @@ public class TableReference implements DeepCopyableWithTransformation<TableRefer
    */
   private final boolean temporary;
 
+  /**
+   * If the table exists in another database then it may be possible to access it with an appropriate database link
+   * (not supported by some database platforms)
+   */
+  private final String dblink;
+

```

Comment:
Why are you initialising this with empty strings intead of nulls?

Note to self: No schemaName is denoted as null, but no alias is denoted as empty string.
So we have a precedence either way, I have not looked into why empty strings too much.

Reply:
no specific reason for that :roll_eyes: do we want to change that to `null` ?